contracts/SavingAccount.sol

```diff
@@ -74,6 +75,13 @@ contract SavingAccount is Initializable, InitializableReentrancyGuard, Constant,
         }
     }
 
+    function initFINAddress() public initializer {
+        if (FIN_ADDR == address(0x0)){
+            FIN_ADDR = address(0x054f76beED60AB6dBEb23502178C52d6C5dEbE40);
+        }
+    }
```

Comment:
Write upgrade tests for this function. Means when we upgrade the `SavingAccount` then :
- We should call this function and this should set the correct FIN address. 
- Also, when you call this function again, nothing should heppen.

Reply:
Added the test for the same